arelle/ModelDocument.py

```diff
@@ -1304,6 +1304,11 @@ def contextDiscover(self, modelContext, targetModelXbrl=None) -> None:
                                     sElt.targetModelXbrl = targetModelXbrl
                                     if hasattr(sElt, "_dimension") and sElt._dimension is None:
                                         del sElt._dimension
+                                    if hasattr(sElt, "_member") and sElt._member is None:
+                                        del sElt._member
+                                    for typedMem in sElt.iterchildren():
+                                        if isinstance(typedMem, ModelObject):  # skip comment and processing nodes
```

Comment:
Out of curiosity, do we need to reset `xValid`, `xValue`, or `xAttributes` on typed dimension elements so that they will be revalidated by the secondary iXDS, or where do those get reset?

Reply:
Reworked for this: see !4851bfb1